src/tools/clippy/tests/ui-toml/min_rust_version/min_rust_version.rs

```diff
@@ -31,10 +31,7 @@ fn option_as_ref_deref() {
 }
 
 fn match_like_matches() {
-    let _y = match Some(5) {
-        Some(0) => true,
-        _ => false,
-    };
+    let _y = matches!(Some(5), Some(0));
```

Comment:
This test should stay as it currently is, since it tests that this does not get linted by Clippy, because of the given configuration.